0004/README.md

```diff
@@ -0,0 +1,416 @@
+
+Nice to have:
+
+1. Distribute provenance verification materials
+
+Out of scope
+
+1. Create a checklist definition of a "secure build platform"
+
+## Prior art
+
+CNCF's Certified Kubernetes Software Conformance -
+[https://www.cncf.io/certification/software-conformance/](https://www.cncf.io/certification/software-conformance/)
+
+## Proposed changes
+
+### Overview
+
+The OpenSSF maintains a SLSA Conformance badge and registry of build platforms
+and auditors that take part in the SLSA Conformance Program. Participating in
+the program allows build platforms to use the SLSA Conformance badge when
+promoting themselves. Any build platform or auditor is free to participate in
+the SLSA Conformance Program provided they are willing to meet its requirements
+and follow its terms.  
+
+The SLSA community provides a security questionnaire for build platforms. This
+questionnaire is the minimum standard for evidence of SLSA conformance.
+Verifiers and auditors are encouraged to seek evidence beyond the questionnaire
+when making their trust determinations. For a draft of the questionnaire, see
+Appendix A.
+
+### Build platform participation
+
+There are two tiers for a build platform to take part in the program:
+self-attestation and third-party audit. Self-attestation is a weaker proof of
+conformance than a third-party audit, and is most useful while bootstrapping the
+conformance program before auditors are available.
+
+#### Tier one: Self-attestation
+
+The self-attestation tier requires that a build platform:
+
+-  Publish answers to the questionnaire on a domain registered to their
+    organization. They may additionally publish further proof of SLSA
+    conformance (e.g. architecture documentation). This information must be
+    updated at least once every twelve months and any time there is an
+    architectural change to the build platform.
+-  Publish any materials needed to verify provenance signed by the build
+    system on a domain registered to their organization.
+-  Publish vulnerability disclosures for all security incidents that
+    potentially impact the integrity of artifacts or provenance generated by
+    the build system.
+-  Sign and follow the SLSA Conformance Program terms.
+
+The OpenSSF operates this tier through a community-maintained repository of
+build platforms. The maintainers of this repository will be drawn from members
+of the SLSA community who are spread across the industry.  
+In order to participate in this tier, build platforms should 
+
+1. Learn about the SLSA Specification and its requirements.
+1. Create a PR for the repository containing
+    1. Name of the build platform
+    1. Build ID
+    1. Desired SLSA levels
+    1. List of URIs that resolve to proof of SLSA conformance
+    1. List of URIs that resolve to materials for verifying provenance
+    1. List of URIs that resolve to the build platform's public
+        vulnerability disclosures
```

Comment:
Are these URIs the `builder.id` or are you trying to associate some other URI with these references?

I think we can simplify this part by grouping all together under a common, something like
> List of URIs that resolve to the self-reported evidence

Reply:
I was thinking other URIs, probably URLs on the build platform's website.

I can't tell from your comment which entries you'd like consolidated. Can you clarify?

Reply:
I was thinking about modifying the list to put all under a common heading. Something like this (i.e. to remove the redundancy around "list of URIs that resolve to"):

* List of URIs that resolve to the self-reported evidence
  * Proof of SLSA conformance
  * Materials for verifying provenance
  * Build platform's public vulnerability disclosures